### ISSUE TITLE
[Bug] react native can't bundle

### ISSUE THREAD

**Issue by gengjiawen** [facebook/metro#167]
In react native, run `./gradlew :RNTester:android:app:installRelease` you can't bundle due to error 
```
SyntaxError: /Users/gengjiawen/FrontEndProjects/react-native/Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js: Support for the experimental syntax 'objectRestSpread' isn't currently enabled (75:5):

  73 | 
  74 |   propTypes: {
> 75 |     ...ViewPropTypes,
     |     ^
  76 |     /**
  77 |      * Determines whether the keyboard gets dismissed in response to a drag.
  78 |      *   - 'none' (the default), drags do not dismiss the keyboard.

Add @babel/plugin-proposal-object-rest-spread (https://git.io/vb4Ss) to the 'plugins' section of your Babel config to enable transformation.

```

**Comment by davidaurelio** [comment]
Thanks for reporting. We noticed this, too. I think I fixed this in 9433e013771a8392fbc491f34413cb9ff5848f63. We will release v0.35.0 today, and I will verify the behaviour with RN.

**Comment by gengjiawen** [comment]
That's quick, thanks.
